app/models/query/business_entity_management.rb

```diff
@@ -0,0 +1,13 @@
+module Query
+  class BusinessEntityManagement
+    def initialize(office)
+      @office = office
+    end
+
+    def jurisdictions
+      Jurisdiction.all.map do |jurisdiction|
+        Views::OfficeBusinessEntityState.new(@office, jurisdiction)
```

Comment:
So, for now the query object would be superfluous?  Because it would only return Jurisdiction.all and then the controller would map the View objects from it?

If you are in tomorrow, maybe it would be better to pair and discuss?